core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
@seanlip No worries, I just threw the changes in. I just commented out the auth requirements and got to the page :) 
Got the changes fixed up. I also changed to use btn-default. 
Attached SS. 
Look good, or should we go back to btn-success?
![image](https://cloud.githubusercontent.com/assets/17284745/25979330/6d72d8b8-368d-11e7-8634-53b2f282981a.png)